doc/ChangeLog

```diff
@@ -1,4 +1,76 @@
 ===============================================================
+Tag name: ctsm5.1.dev1??
+Originator(s): slevis (Samuel Levis, 303-665-1310)
+Date: Fri Jul 29 03:49:06 MDT 2022
+One-line Summary: Introduction of modify_meshes tool for use in I-cases and F-cases
+
+Purpose and description of changes
+----------------------------------
+
+ Merging the modify_meshes tool to main. This tool is a follow-up to the
+ modify_fsurdat tool for I- and F-cases where users are adding land to
+ areas that by default were expected as ocean. In such cases, fsurdat
+ modifications will take effect only if users also modify the mask_mesh
+ file which ultimately determines the landmask used by the CTSM.
+ Details in the tool's README.
+
+ Adding test_sys_mesh_modifier.py to python system tests.
+
+
+Significant changes to scientifically-supported configurations
+--------------------------------------------------------------
+
+Does this tag change answers significantly for any of the following physics configurations?
+(Details of any changes will be given in the "Answer changes" section below.)
+
+    [Put an [X] in the box for any configuration with significant answer changes.]
+
+[ ] clm5_1
+
+[ ] clm5_0
+
+[ ] ctsm5_0-nwp
+
+[ ] clm4_5
+
+
+Bugs fixed or introduced
+------------------------
+CTSM issues fixed (include CTSM Issue #): #1644
+
+Notes of particular relevance for developers:
+---------------------------------------------
+Changes to tests or testing:
+ Adding test_sys_mesh_modifier.py to python system tests. This requires
+ "module load nco" before running "make stest" or "make all" in the /python
```

Comment:
@ekluzek question for when we go over this PR:
Could we include the `module load nco` in py_env_create, so that users won't have to think about it separately? If that's not an option, then do we have other existing infrastructure (eg in cime) that I could point to? @jedwards4b I'm copying you in case you have insight. Currently I run `module load nco` interactivly before running this new ctsm python system test. I would prefer to make the `module load` transparent to the user.

Reply:
Hmmm. I could add that for our NCAR systems. So it would work at NCAR, but fail elsewhere. It's not really part of the CTSM python environment though. So I'd like to run this by people and see what they think.

Reply:
I guess I am confused - are you building tools specific to a given system (eg cheyenne)
modules are different on every HPC system, about the best you can do is provide a README file.

Reply:
@jedwards4b we want this tool to work everywhere you might run CTSM. So yes you are right we could maybe add the module load for a few NCAR systems, but otherwise we'll have to rely on the README file you talk about, and some error checking that fails if NCO isn't available on your system.

Reply:
- [ ] Add a README that talks about the dependence of NCO
- [ ] Add error checking so that the script fails with a clear error if NCO isn't in the users path

Reply:
OK, the dependence on NCO is actually only for the test, but we still want to have this documented and easily seen. So resolving this.